src/Fix64.cs

```diff
@@ -359,6 +368,94 @@ public partial struct Fix64 : IEquatable<Fix64>, IComparable<Fix64> {
             return x.m_rawValue <= y.m_rawValue;
         }
 
+        public static Fix64 Exp(Fix64 x)
+        {
+            if (x.RawValue == 0) return One;
+            if (x == One) return E;
+            if (x >= LnMax) return MaxValue;
```

Comment:
What are the checks against LnMax and LnMin for? The tests still pass without. Optimization?

Reply:
Yes. I figured if we know precisely where the argument range of the function ends, might as well test for it, but I guess if the early bailout test rarely succeeds it's a waste of cycles. I'm fine with removing it, your call.

Reply:
It's a good optimization I think, just wanted to make sure.